app/components/Button/ButtonStyled.js

```diff
@@ -0,0 +1,66 @@
+import AddText from '../AddText';
+
+const ButtonStyled = AddText.extend`
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  padding: 9px 16px;
+  border-radius: 3px;
+  min-width: 92px;
+  box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.4);
+  background: #f76f60;
+  height: 40px;
+
+  position: relative;
+  overflow: hidden;
+
+  .chevron {
+    color: #fff;
+  }
+
+  :disabled {
+    border: 1px solid #bdbdbd;
```

Comment:
Value #bdbdbd appear in multiple places. Think if any colors, text size or any other parameters are shared for components. They can be moved to Js file with constant variables for theming.

Reply:
decided